deploy-board/deploy_board/templates/host_side_panel.html

```diff
@@ -28,6 +28,10 @@ <h4 class="modal-title">Host Termination Confirm</h4>
                             <strong>Warning!</strong> This host is unreachable. It cannot be gracefully shut down.
                         </div>
                     {% endif %}
+
+                    <input type="checkbox" id="replace_host" name="replaceHost" checked> Check to replace host (Uncheck to to decrement desired capacity)
+                    <br><em>Note replacement will only work if the host belongs to an auto scaling group.</em>
+                    <br><em>If you choose to not replace host, it can't be guarenteed that the auto scaling group's min and max sizes will keep identical.</em>
```

Comment:
```suggestion
                    <br><em>If you choose to not replace host, it can't be guaranteed that the auto scaling group's min and max sizes will keep identical.</em>
```